dsl-reference.md

```diff
@@ -1416,6 +1417,36 @@ from:
 to: '.petList += [ . ]'
 ```
 
+### Export
+
+Certain task needs to set the context of the workflow using the task output for later usage. User set the content of the context through a runtime expression. The result of the expression is the new value of the context. If user want to merge the new data into the current context, the context is accessible within the expression scope: `.` if using jq. 
```

Comment:
Speaking of the `.` is irrelevant, because it means the same than anywhere else: its the expression's input.

I think you should leave `jq` out of it. I would remove the export section, and just describe the property with something like:

A [`runtime expression`](#runtime-expressions) evaluated again the workflow/task context data to update or replace it.

Reply:
I think is critical to clarify that the result of the expression is the new value of the context. And therefore, if you want to use the existing context, you are supposed to use `.`

Reply:
> I think is critical to clarify that the result of the expression is the new value of the context. And therefore, if you want to use the existing context, you are supposed to use .

I disagree. `.` is irrelevant because it is tightly bound to `jq`. You should not even mention the latter, all you are going to achieve is confusing users who use another expression language.

The sentence I proposed clearly indicate that the expression should be run AGAINST the context, and that it is used to update or replace it. I can change it so that the word 'output' clearly appears, but more than that...

Reply:
Please check the repharse. I mention jq as a possible example.

Reply:
> If user want to merge the new data into the current context, the context is accessible within the expression scope: `.` if using jq. 

The rephrase still is confusing to me. Of course I can merge or whatever if the context becomes whatever I output.

Reply:
yes, but is important to indicate the writer how to do that using jq. I will remove the JQ reference and leave the example

Reply:
Done, jq is only used in the example

Reply:
Ok, another try

Reply:
Perfect! Thank you man!

Reply:
So the input to the `export.to/as` expression is the context (makes sense) but then the output would need to be available as an expression argument via `$output` right?

Reply:
Yes, exactly, we need to add ouput and it should imo be part of that PR indeed

Reply:
Can you do that too, @fjtirado ?